.ci/.e2e-tests-nightly.yaml

```
@@ -0,0 +1,61 @@
+---
+SUITES:
+  - suite: "helm"
+    provider: "docker"
+    scenarios:
+      - name: "APM Server"
+        tags: "apm-server"
+        platforms: ["debian_amd64"]
+      - name: "Filebeat"
+        tags: "filebeat"
+        platforms: ["debian_amd64"]
+      - name: "Metricbeat"
+        tags: "metricbeat"
+        platforms: ["debian_amd64"]
+  - suite: "fleet"
+    scenarios:
+      - name: "Fleet"
+        tags: "fleet_mode"
+        platforms: ["centos8_arm64", "centos8_amd64", "debian_arm64", "debian_amd64", "sles15"]
+      - name: "Integrations"
+        tags: "integrations"
+        platforms: ["debian_arm64", "debian_amd64", "sles15"]
+      - name: "APM Integration"
+        tags: "apm_server"
+        platforms: ["debian_amd64"]
+      - name: "Linux Integration"
+        tags: "linux_integration"
+        platforms: ["debian_arm64", "debian_amd64", "sles15"]
+      # - name: "Permissions Output change"
+      #   tags: "permission_change"
+      #   platforms: ["debian_arm64", "debian_amd64", "sles15"]
+      - name: "System Integration"
+        tags: "system_integration"
+        platforms: ["debian_arm64", "debian_amd64", "sles15"]
+      # - name: "Stand-alone"
+      #   tags: "stand_alone_mode"
+      #   platforms: ["fleet_elastic_pkg"]
```

Because it's a new file, I think we can remove comments:
```suggestion
      - name: "System Integration"
        tags: "system_integration"
        platforms: ["debian_arm64", "debian_amd64", "sles15"]
```